keps/sig-auth/20190916-noderestriction-pods.md

```diff
@@ -200,22 +202,25 @@ pods. However, there are a few labels that are commonly applied to system addons
 The `k8s-app` label is used to match controllers for system components, and therefore should be
 explicitly disallowed.
 
-It is not clear how the `tier` and `component` labels are consumed, but I recommend uses be migrated
-to `unrestricted.node.kubernetes.io/component` and `unrestricted.node.kubernetes.io/tier`, rather
-than special casing the existing labels.
-
-Additionally, nodes will be restricted from making any label changes through `pod/status`
-updates. As the kubelet doesn't make any label updates through this request, this change  will be
-rolled out without a feature gate.
+`kubeadm` should be modified to whitelist the `component` and `tier` labels, or potentially drop
```

Comment:
This is very reasonable, but you'll need to make them aware.

Reply:
@tallclair reading the KEP does this mean that kubeadm created cluster need to:

annotate the namespace like so:
```
$ kubectl annotate namespaces kube-system node.kubernetes.io/mirror.allowed-label-keys="component,tier"
```

before the `MirrorPodNodeRestriction` gate becomes `true` by default?

@kubernetes/sig-cluster-lifecycle

Reply:
Yes, the options are to whitelist those keys, or stop labeling static pods.